pkg/rancher-desktop/main/diagnostics/types.ts

```diff
@@ -1,4 +1,5 @@
 export enum DiagnosticsCategory {
+  ContainerRuntime = 'Container Runtime',
```

Comment:
Should this be "Container Engine"? We try to use "container engine" for moby or containerd, and "container runtime" for runc.